src/material/schematics/ng-generate/theme-color/README.md

```diff
@@ -34,11 +34,46 @@ html {
 }
 ```
 
-High contrast override theme mixins are also generated in the file if specified
-(`high-contrast-light-theme-overrides` and `high-contrast-dark-theme-overrides`). These mixins
+## High contrast override mixins
+High contrast override theme mixins are also generated in the file if specified. These mixins
 override the system level variables with high contrast equivalent values from your theme. This is
 helpful for users who prefer more contrastful colors for either preference or accessibility reasons.
 
+### Creating one theme for light and dark mode
+As of v19, the `theme` mixin can create one theme that detects and adapts to a user if they have
```

Comment:
We can work on this wording as we get closer to release. We'll sync up the messaging between the guide and this